docs/guides/max-execution-time.mdx

```diff
@@ -46,13 +46,19 @@ For example, if you specify `max_execution_time=5000` (approximately 83 minutes)
 
 ## Session maximum execution time
 
-When a session is started, it is assigned a maximum session timeout value. After this timeout is reached, the session is terminated, any jobs that are already running continue running, and any queued jobs that remain in the session are put into a failed state. For instructions to set the session maximum time, see [Specify the session length](./run-jobs-session#specify-length).
+When a session is started, it is assigned a maximum time to live (TTL) value. After this TTL is reached, the session is terminated, any jobs that are already running continue running, and any queued jobs that remain in the session are put into a failed state. For instructions to set the session maximum time, see [Specify the session length](./run-jobs-session#specify-length).
+
+Sessions also have an interactive time to live (ITTL) value between jobs that cannot be configured. If no session jobs are queued within that window, the session is temporarily deactivated.
+
+To determine the maximum TTL and ITTL values for a session, use the [`session.details()` method](run-jobs-session#session-details) and look for the `interactive_timeout` and `max_time` values, respectively.
 
 ## Batch maximum execution time
```

Comment:
Just notice the title of this page is _Maximum execution time for a Qiskit Runtime job or session_. Instead of adding Batch to that sentence, how about _Maximum execution time for different execution mode_ ?